modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/AttachmentUtil.java

```diff
@@ -259,13 +264,128 @@ public static CPAttachmentFileEntry addOrUpdateCPAttachmentFileEntry(
 			GetterUtil.get(attachmentUrl.getGalleryEnabled(), true),
 			getTitleMap(null, attachmentUrl.getTitle()),
 			_getJSON(
-				cpDefinitionOptionRelService, cpDefinitionOptionValueRelService,
-				cpOptionService, attachmentUrl.getOptions(), classPK,
+				null, cpDefinitionOptionRelService,
+				cpDefinitionOptionValueRelService, cpOptionService,
+				attachmentUrl.getOptions(), classPK,
 				serviceContext.getCompanyId()),
 			GetterUtil.getDouble(attachmentUrl.getPriority()), type,
 			serviceContext);
 	}
 
+	public static CPAttachmentFileEntry addOrUpdateCPAttachmentFileEntry(
```

Comment:
why two methods? If needed please let the other one call this one with the defaults (or null) ... don't duplicate the code

Reply:
and the parameters I think should be sorted